xbmc/interfaces/python/XBPython.cpp

```diff
@@ -45,7 +45,7 @@
 #include "interfaces/legacy/AddonUtils.h"
 #include "interfaces/python/AddonPythonInvoker.h"
 #include "interfaces/python/PythonInvoker.h"
-
+PyThreadState* savestate;
```

Comment:
Since the functions savestate is used in don't share a common class (neither heirarchial), the option other than gloabl variable would be to define the variable static in CPythonInvoker